client/scenes/stream_tracking.cpp

```diff
@@ -243,6 +244,12 @@ void scenes::stream::tracking()
 						t.resume();
 					}
 
+					if (application::get_fb_face_tracking2_supported())
+					{
+						application::get_fb_face_tracker2().get_weights(t0 + Δt, &fb_face2);
+						network_session->send_stream(fb_face2);
```

Comment:
Timer should be paused when sending, I guess manually doing this everytime doesn't scale, we can make a macro so that future additions won't miss it.

Rationale: socket sometimes block (queue full? radio error?), and mess up time measurement. On some runtimes, locating spaces is heavy on CPU time and it is not possible to run the loop at 1kHz rate. Lowering the sampling rate helps with thermals and is a bit better. We don't want to lower the rate for occasional network spikes.